astroquery/ipac/irsa/tests/test_irsa.py

```diff
@@ -120,9 +121,11 @@ def test_query_region_box(coordinates, patch_get):
     assert isinstance(result, Table)
 
 
-poly1 = [coord.SkyCoord(ra=10.1, dec=10.1, unit=(u.deg, u.deg)),
-         coord.SkyCoord(ra=10.0, dec=10.1, unit=(u.deg, u.deg)),
-         coord.SkyCoord(ra=10.0, dec=10.0, unit=(u.deg, u.deg))]
+poly1 = [
+    SkyCoord(ra=10.1 * u.deg, dec=10.1 * u.deg),
+    SkyCoord(ra=10.0 * u.deg, dec=10.1 * u.deg),
+    SkyCoord(ra=10.0 * u.deg, dec=10.0 * u.deg),
+]
```

Comment:
Keep the style, the additional 2 lines don't add anyting
```suggestion
poly1 = [SkyCoord(ra=10.1 * u.deg, dec=10.1 * u.deg),
         SkyCoord(ra=10.0 * u.deg, dec=10.1 * u.deg),
         SkyCoord(ra=10.0 * u.deg, dec=10.0 * u.deg)]
```